### ISSUE TITLE
Alias "Movers" to "Moving Company Office"

### ISSUE THREAD

**Issue by michaelblyons** [openstreetmap/id-tagging-schema#124]
I typed too many letters (i.e. `mov` works, but `movers` doesn't), and eventually consulted the Wiki to find the appropriate tagging. Can you add a match for `movers?` to point people at `office=moving_company`?

**Comment by matkoniecz** [comment]
Which other values also need this? It would be nice to implement it in one batch.

---------------------

It should be solved by adding alias for "Moving Company Office" in presets.

I found an example commit in https://github.com/openstreetmap/id-tagging-schema/commit/942e4ce508b02ba3ea5ab516620181b5326e8a6a

Please, ping me after  #116 is accepted or rejected and I will implement this.

**Comment by michaelblyons** [comment]
> Which other values also need this? It would be nice to implement it in one batch.

Other aliases for Moving Company Office? It's possible that some places use "hauling" or "haulers" for that, but I do not know. In my dialect, "haul" usually implies trash or very large freight.

If "hauling/haulers" matches few enough items, we could add it as an alias for `office=logistics`, `office=moving_company`, _and_ Trash Removal (can't find a tag in the Wiki&mdash;maybe `office=moving_company` is already the closest thing?).

> It should be solved by adding alias for "Moving Company Office" in presets.

Agreed. 👍 

> Please, ping me after #116 is accepted or rejected and I will implement this.

Okay. I'm not savvy enough to understand how #116 is related, but I'm happy to wait until there's a decision on it.

**Comment by matkoniecz** [comment]
> Okay. I'm not savvy enough to understand how #116 is related, but I'm happy to wait until there's a decision on it.

It is a pull request that I opened, and I want to wait for processing it before opening a new one.

**Comment by michaelblyons** [comment]
I see why! It took 11 months to be merged.

**Comment by mbrzakovic** [comment]
Apologies for waiting on this.
I agree with @matkoniecz, I recommend this issue be resolved by adding appropriate terms + aliases to existing preset.

**Comment by matkoniecz** [comment]
> Ping

Pong.

#265 now exists

**Comment by matkoniecz** [comment]
> If "hauling/haulers" matches few enough items, we could add it as an alias for office=logistics, office=moving_company, and Trash Removal (can't find a tag in the Wiki—maybe office=moving_company is already the closest thing?).

Yes, I agree. I will not add them blindly as I never encountered and used them like this (but I am NOT a native speaker, do not treat it as indicator of a bad idea). If  other aliases are needed then direct edit (it is relatively simple) or new issue would be likely a better idea.

This specific issue will be fixed by #265
